### DISCUSSION TITLE
Correlation in the Integral Wilber-Coombs Matrix

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#394] [Research]
The Wilber-Coombs matrix (or lattice) maps states (waking up) against stages (growing up):

<img src="https://lifeitself.org/assets/images/wilber-coombs-lattice-4x7-1.jpg" height="300" />

### Key aspect: two *independent* dimensions of development

A key aspect of the Wilber-Coombs matrix is that any location of the matrix is allowed: i.e. states and stages are independent.  Put another way: inner development (aka ontological growth) is, at minimum, two dimensional with independent dimensions of states ("waking up") and stages ("growing up").[^1][^2]

[^1]: Of course, development is even more multi-dimensional but this was a big advance from linear developmental models that stacked waking up on top of growing up (as in early Wilber in the 70s).

### But are the two dimensions independent?

Whilst these two dimensions may be distinct, they could still be **correlated**. That is, development pathways require advance along both dimensions, at least to an extent: i.e. to grow up to a certain point you may need to have "woken up" to a certain extent or vice versa. Conversely, it is difficult or impossible to be "teal" and to still be in "gross" waking up state. 

Here's a very crude sketch of a potential correlation matrix version of Wilber-Coombs (colored by heat-mapping) showing how movement along the "waking up" and the "growing up" dimension are correlated.

<img src="https://user-images.githubusercontent.com/180658/228456690-47a7a195-db59-40f4-ad49-8ee67d2e44e9.png" height="400" />

[Source spreadsheet](https://docs.google.com/spreadsheets/d/1PhZ3ax2FbNlFYpUIClniOP8ibQ4hZPedu7-5tLdJObc/edit#gid=0)

## More Discussion

- The overall separation has been questioned. In particular, Terri O'Fallon has continued to argue that you can collapse these back onto uni-dimensional developmental line. See e.g. Collapse of Wilber-Coombs Matrix (2010)

[^2]: A second important point, not emphasized here so much, is that states are interpreted through stages.

**Comment by baouroux** [comment]
I've not seen this image before... Super interesting to be reminded of the nuances within the Wilber-Coombs mapping. And wouldn't it be interesting to have some Qual /Quant data on this.

**Reply by rufuspollock** [reply]
Quant data on this? You mean some actual studies of people at particular locations in the "matrix"? Dream on 😉 - i say that is i don't really know of any methodology for rigorously assessing stages. And the best for states would be Jeffery Martin's work which only happened in the 2010s and which doesn't have a publicly available assessment system.

That said, there may be some good qual data and examples. The classic Wilber one is Ram Dass' brother who he talks about regularly on podcasts (seriously awakened in Amber)

**Reply by asimong** [reply]
Nor, to my knowledge, has Kegan published any assessment instrument. are those examples from podcasts documented in any way?

**Reply by rufuspollock** [reply]
> I've not seen this image before...

I created it a while back when having a debate about this very question i.e. was there correlation, or, more specifically: does waking up support growing up and vice versa. e.g. does having a sense that "you are not (just) your thoughts" help in, say, perspective taking.

**Comment by asimong** [comment]
For sure, "dream on" if you're looking for pre-existing studies. But, on the other hand, I like very much the thought experiment of consdiering what would it take to create reliable tools to place people / groups / cultures at one or more positions on these scales — that would at least open the possibility that the correlations were more soundly based than someone just thinking them through. I'd be looking at least for inter-rater reliability here, of which there was plenty in William G Perry's work, predecessor of Robert Kegan.

The other thing I'd really like to dream on about is how we (who?) can support people's chosen self-development in these various dimensions. Yes, we're trying out the things we believe are helpful, but what evidence is there that they actually help across the board? In this kind of field, I find anecdotal evidence very frustrating, because of course you will get some people who are helped. What is is that people who are helped share with the narrators of the narratives?

To me, some of it is still too much the conceptual scheme of a very few people, taken as authorities (in a very 20th Century way — which stage is that then???) and will continue to be until there's at least ontological commoning, and then I would hope some experimental as well as experiential evidence. A very useful basis I imagine would be a large set of real biographies (even autobiographies) that are relatable to developmental stages etc.

Without at least convincing narratives, if not real experiments and reliable rating tools, I'm inclined to see much of this kind of work as personal healing narratives. Which are very interesting in themselves, but hardly generalisable. They appeal to those whose internal healing narratives are aligned. And not to those who aren't.

**Reply by rufuspollock** [reply]
> For sure, "dream on" if you're looking for pre-existing studies. But, on the other hand, I like very much the thought experiment of consdiering what would it take to create reliable tools to place people / groups / cultures at one or more positions on these scales — that would at least open the possibility that the correlations were more soundly based than someone just thinking them through. I'd be looking at least for inter-rater reliability here, of which there was plenty in William G Perry's work, predecessor of Robert Kegan.

👍👍 big plus one.

> The other thing I'd really like to dream on about is how we (who?) can support people's chosen self-development in these various dimensions. Yes, we're trying out the things we believe are helpful, but what evidence is there that they actually help across the board? In this kind of field, I find anecdotal evidence very frustrating, because of course you will get some people who are helped. What is is that people who are helped share with the narrators of the narratives?

Yes, first you want a "map" i.e. a "theory" of ontological development (perhaps both personal and collective as they are distinct though interwoven). Then you want a bus (means of transport) to move around the map e.g.  protocols, practices, etc.

## 🗺️ + 🚌

Or as @lkavanagh30 put it in a slightly different context: maps and rafts!

## 🗺️ + 🚣

**Comment by baouroux** [comment]
Just catching up on this thread, thanks Rufus and Simon for great
contributions - yes, dreaming is the occupational hazard here, sucked into
the matrix!

Look forward to hearing more Simon on your ontological commoning - a
fascinating framing.

1. Stage Development Relationship with States:
Continuing with the Coombs-Wilber matrix, that's so interesting Rufus that
you put that graphic together. You don't need to answer this here, but to
bookmark it for a group discussion when the occasion arises, I
was very interested (and hence my data question!) as to, on what evidence
can we draw on, to say that the deeper states are accessed at higher stage
levels?

That brings us perhaps back to what is a definition of a stage (up for
debate).

I've heard discussed in a couple of forums how access to states is a key
dynamic in the movement between stages, but it goes back a while.

In more everyday contexts, I'm interested in the 'necessary' access to
emotional states in therapeutic settings; from references such as Eugine
Gendlin's 'Focusing' therapeutic dialogue, to Kazimierz Dąbrowski's
'Positive Disintegration' theory. The transition between stages could be
inferred to include a relinquishing of previous structures of meaning
making, that is an emotionally charged process. How might emotions (and
embodiment) and states relate?

In terms of dreaming on :) with the qual and quant data, I had in mind how
there have been such extensive recent research projects of meditative
states in the labs, that if you were to combine such physiological outputs
with developmental stages assessment tools, you could perhaps bring them
together?

As an anecdote of measuring states, have you seen Ken drop his brain waves
here?!! Ken Wilber measuring on an ESG his meditation
<https://www.youtube.com/watch?v=LFFMtq5g8N4&ab_channel=IntegralLife>

2. Biography and Stages:
Picking up on Simon's references, I would love to go deeper on personal
journeys and biographies...contrasting with my certification the Lectica
developmental tool (based principally on Commons and Fischer's hierarchical
cognitive complexity), there is of course the Washington University
Sentence Completion Test models (great article here from Tom Murray on the
evolutions with that
<http://integralleadershipreview.com/15642-sentence-completion-assessments-for-ego-development-meaning-making-and-wisdom-maturity-including-stages-overview-and-summary-this-article-began-as-a-series-of-short-white-papers-providing-various/>)
where the stage labels refer more to 'Wisdom' and 'Ego', through to Kegan's
Subject-Object Interview (SOI).

I've been trying to build a library of biographies, with the particular
silent, unconditional witnessing practice on film, that captures the
personal journey from breakdown to breakthrough....herewith a couple of
examples (apologies, these are long edits, I've not yet created shorter
extract edits):

   - https://www.interiortruth.com/sue-cooper
   - https://www.interiortruth.com/shannon-mcarthur

The film subjects in both cases stepped away from the witnessing saying
they have no idea how long they spoke for, or what they said! This is my
own quest for gathering 'data'!!!

With regards biography and Stages, herewith
<https://drive.google.com/file/d/1QgNzMHsw8NgzUpQYdYMi-k8FQ5Bzhuzq/view?usp=sharing>
an ebook by a veteran story worker Joe Lambert (including in theatre) that
weaves in the developmental stages lens...a rare voice that I have come
across, bringing the narratives and stages domains together.

To be continued I hope on the research calls...

Cheers,
John

**Reply by rufuspollock** [reply]
>  I was very interested (and hence my data question!) as to, on what evidence
can we draw on, to say that the deeper states are accessed at higher stage
levels?

To clarify: the correlation matrix was trying to indicate the other way round: i.e. higher stages only accessed with higher states (in correlation matrix higher states are available at all stages, but for some stages e.g. Teal and up you would need to be in higher stages - e.g. at least subtle -  to access them)

**Comment by baouroux** [comment]
Thanks, yes, I was using unclear vocabulary... and had used 'deeper' as a
synonym for higher / causal states.

**Comment by baouroux** [comment]
My neurons just catching up, right, yes so accessing the higher states
thanks to being at the higher stage.

Wow, quite a revolutionary addition to stage definition and no stage
assessment that I know of takes into consideration states.

I'm excited to hear more on this...

I use the reference to stage assessment with also a personal advocation
that stage theory moves from measuring people being 'at' a particular stage
in the 3rd person, to stage theory being more of a 2nd person dialogue
scaffold, where the stages help in sensemaking in more situational and
context specific instances - in large part due to the seemingly infinite
complexity across the different developmental lines.
